test/util.js

```diff
@@ -0,0 +1,41 @@
+const shell = require('shelljs');
+const path = require('path');
+const rootDir = process.cwd();
+
+shell.config.silent = true;
+
+// shelljs.grep wrapper
+// @param {RegExp} pattern
+// @param {string} fileName
+// @returns {boolean} true if pattern has matches in file
+function grep(pattern, fileName) {
+  const output = shell.grep(pattern, fileName);
+  // output.code is always 0 regardless of matched/unmatched patterns
+  // so need to test output.stdout
+  // https://github.com/jaredpalmer/tsdx/pull/525#discussion_r395571779
+  return Boolean(output.stdout.match(pattern));
+}
```

Comment:
ehhhh this should probably be in a separate file because all the functions here are solely fixture related. This file was originally in `fixtures/util.js` so maybe it should be in `utils/fixture.js` instead and this in a more generic `utils/misc.js` or something.

Reply:
yep this was put into the same `utils/shell.js` from #621's recent changes
and `test/util.js` was moved to `utils/fixture.js`